docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -1,91 +1,75 @@
 [id='{context}-pro-running-the-demo-repositories']
-= Running the {PROJECT_NAME} Applications on a Local Machine
+= Running the {PROJECT_NAME} Demonstration Environment
```

Comment:
I think we downgraded this statement:
Running on local machine is clearly stating the purpose.

"Demonstration Environment" - it's not common term. It carries less info than just local machine. 
As person who read this for the first time I asked myself - what "Demostration environment" is

Reply:
I wasn't sure about 'demo environment' either. what about "Running the Raincatcher Applications Locally"? or just revert back to the original?

Reply:
😆 Running the Raincatcher Applications Locally - were used before and was changed in the review process. Let's revert.